azure-core/azure/core/configuration.py

```diff
@@ -24,9 +24,11 @@
 #
 # --------------------------------------------------------------------------
 
+#pylint: disable=too-few-public-methods
 
-class Configuration(object):
```

Comment:
The `(object)` needs to be here, because it's needed for 2.7 compatibility.
If you're getting pylint errors for unnecessary object inheritance - we should add an global exception to this in the pylint.rc file.
Which reminds me.... are you running pylint with the pylint.rc file in the root of the repo?

Reply:
Yup. I'm using it.